frontend/src/components/common/Table.tsx

```diff
@@ -20,39 +21,59 @@ interface TableProps {
 }
 
 export const Table = ({ headers, rows }: TableProps): React.ReactElement => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const itemsPerPage = 8; // can edit this to show how many itemsperpage we want
```

Comment:
creds to @joyce-shi for this comment:

would you be able to remove the extraneous comments here and below? in general, it's best practice not to have comments duplicate / describe the code functionality. instead, the code should be clear enough to speak for itself!

you can read more on this here: https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/